src/components/LanguageSelectWithGetStarted.astro

```diff
@@ -0,0 +1,37 @@
+---
+import Default from '@astrojs/starlight/components/LanguageSelect.astro';
+
+// Get the current page route
+const route = Astro.locals.starlightRoute;
+const isSnowflakePage = route.id.startsWith('snowflake');
+
+// Build the URL with conditional parameter
+const getStartedUrl = isSnowflakePage
+  ? 'https://app.localstack.cloud/sign-up?emulator=snowflake'
+  : 'https://app.localstack.cloud/sign-up';
```

Comment:
```suggestion
const isSnowflakePage = route.id.startsWith('snowflake');

// Build the URL with conditional parameter
const getStartedUrl = isSnowflakePage
  ? 'https://app.localstack.cloud/sign-up?emulator=snowflake'
  : 'https://app.localstack.cloud/sign-up';
```

In this case, should we remove these lines @remotesynth ?

Reply:
I tweaked the lines but left them in. The URL variable doesn't really do anything it seems at the moment but I think it's worth differentiating which docs they came from anyway.